lib/member_page.rb

```diff
@@ -0,0 +1,32 @@
+#!/bin/env ruby
```

Comment:
you don't need a shebang line here (or in anything you're not actually going to directly run)